src/microbe_stage/editor/ToleranceInfo.tscn

```diff
@@ -0,0 +1,65 @@
+[gd_scene load_steps=3 format=3 uid="uid://dccswprlmgtel"]
```

Comment:
I think everything should have mouse ignore / pass here because right now this adds a "break" area between the value slider and the current value readout where the relevant stat tooltip is not shown due to this messing with the mouse passing.